templates/named.conf.options.erb

```diff
@@ -62,6 +62,12 @@ check-names slave <%= @check_names_slave %>;
 check-names response <%= @check_names_response %>;
 <% end -%>
 
+<% if @statistic_channel_ip and @statistic_channel_ip -%>
```

Comment:
Na, I don't think we need to test the validation. I'm mostly concerned about the template string values that end up on disk.